sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management/src/main/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+feature-management:
```

Comment:
Missed a file.

Refs: https://github.com/Azure/azure-sdk-for-java/pull/18916/files#diff-c1757f45b3b5c0a689db0a125951dbeea7fd7416d62bfd6663ae45836758542bL1